package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/abac",
-  "version": "2.0.5",
```

Comment:
Since the exported type names changed would it be safer to do a major version bump?

Reply:
I thought about this and since the functionality didn't change I was leaning toward minor version bump. The typescript projects will be forced to switch to the new functions but I don't think that will be a big issue. I could go either way on this I guess.

Reply:
I just went ahead and made this a major version bump.